### PR TITLE
added my info to student section

### PR DESCRIPTION
**Pull request by yeezhi**
added my info to students section

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://yeezhi.github.io/gci15.fossasia.org

**Comment by jigyasa-grover** [comment]
@yeezhi Travis build failed. 

Build error: `jekyll 2.4.0 | Error:  (/home/travis/build/fossasia/gci15.fossasia.org/_data/students.yml): mapping values are not allowed in this context at line 288 column 9`

And may I ask the addition of file `img/students/Thumbs.db` ?

File: _data/students.yml [hunk 1]
```diff
@@ -282,4 +282,11 @@
   image: JLeow.png
   facebook: jerome.leow.5
   twitter: JLeow00
-  bio: Ukulele is love, ukulele is life
\ No newline at end of file
+  bio: Ukulele is love, ukulele is life
+
+- name:Leow Yee Zhi
```
Comment (roonyh) [inline _data/students.yml]: You need a space after `:`

**Comment by yeezhi** [comment]
fixed it

**Comment by roonyh** [comment]
Yep. Good job! Thanks for keeping it to a single commit :+1:

**Pull request merged**